corehq/motech/repeaters/models.py

```diff
@@ -356,12 +361,29 @@ def plaintext_password(self):
             return b64_aes_decrypt(ciphertext)
         return self.password
 
-    def get_auth(self):
+    def get_auth_manager(self):
```

Comment:
You could make a standalone `get_auth_manager` function called by both of these, though that wouldn't exclude the OATH versions as this method does.  Perhaps in the future.

Reply:
This method is temporary. In a follow-up PR (branch waiting in the wings) Repeaters will use ConnectionSettings instead of having their own duplicated data & methods.